### ISSUE TITLE
Custom issue number formats

### ISSUE THREAD

**Issue by TheEvilDev** [conventional-changelog/standard-version#426]
We have a separate issue tracker at my company, and our issue numbers are in the format similar to "ABC-123" rather than github "#123". Any way we can provide a regex or something for the issue number format for substitution?

**Comment by bcoe** [comment]
@tommywo we landed this functionality recently right? (addressing @markmsmith's feature request).

**Comment by tommywo** [comment]
it's added to conventional-changelog-conventionalcommits and it can be used with semantic-release, as it allows to set issuePrefixes in parserOpts, but unfortunately i didn't notice not possible to set it in standard-version.

**Comment by bdellegrazie** [comment]
@TheEvilDev / @tommywo: current master works setting issuePrefixes in .versionrc.json (or whatever)  - as per your recently added test. If a release is pushed this issue would be resolved

**Comment by Hagith** [comment]
I looking forward to use custom issue prefix. When can we expect the next release?

PS: Great tool, thanks for your work!

**Comment by bcoe** [comment]
@tommywo anything blocking a release of `master`, I'm not 100% up-to-date with recent work on the project.

**Comment by tommywo** [comment]
No, I would release it before but i don't have permission on npm ¯\_(ツ)_/¯

**Comment by bcoe** [comment]
@tommywo sorry about that, I thought it was part of the org; I just sent an invite your way I think?

**Comment by tommywo** [comment]
I've published 7.0.1@next this morning.

**Comment by bcoe** [comment]
@tommywo thank you 👏
